modules/swagger-codegen/src/main/resources/JavaSpring/apiController.mustache

```diff
@@ -77,5 +78,12 @@ public class {{classname}}Controller implements {{classname}} {
 
 {{/operation}}
 {{/jdk8-no-delegate}}
+    private ObjectMapper getObjectMapperByContentType(String mediaType){
```

Comment:
@joymarielim would be good to add a docstring to explain what this function does